compare/commands/compare/compare.py

```diff
@@ -1,8 +1,20 @@
+from typing import Union
+
+from shared.reports.resources import Report
+
 from codecov.commands.base import BaseCommand
+from services.comparison import Comparison, ComparisonReport, PullRequestComparison
 
 from .interactors.fetch_impacted_files import FetchImpactedFiles
 
 
 class CompareCommands(BaseCommand):
-    def fetch_impacted_files(self, comparsion, filters):
-        return self.get_interactor(FetchImpactedFiles).execute(comparsion, filters)
+    def fetch_impacted_files(
+        self,
+        comparison_report: ComparisonReport,
+        comparison: Union[PullRequestComparison, Comparison],
```

Comment:
Just `Comparison` is fine here since `PullRequestComparison` inherits from `Comparison` and we're not using anything specific to `PullRequestComparison`.